third_party/pocketfft/workspace.bzl

```diff
@@ -19,11 +19,11 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 def repo():
     http_archive(
         name = "pocketfft",
```

Comment:
Would it make sense to add ducc0 as a parallel dependency to pocketfft rather than replacing pocketfft for the moment?

I'm concerned that it might take a while to enable C++17 by default, but I don't want that to hold up your PR. One way we could manage that would be to add ducc0 as an alternative to pocketfft and select between the two depending on whether a C++17 compiler is available.

Reply:
I'm not in a hurry to have this merged; we can just wait until C++17 is acceptable. My main goal was to see how big the necessary adjustments will be (surprisingly small, it seems!).
I don't know how to add this as a parallel dependency, but if you help me do that, it's also perfectly fine with me.